Flaking Test TestResolvedTs

### Which jobs are flaking?

https://ci.pingcap.net/blue/organizations/jenkins/ut-check/detail/ut-check/17074/pipeline/

### Which test(s) are flaking?

TestResolvedTs

### Jenkins logs or GitHub Actions link

```log
[2022-06-14T01:22:07.009Z] === Failed
[2022-06-14T01:22:07.009Z] === FAIL: cdc/model TestResolvedTs (0.00s)
[2022-06-14T01:22:07.009Z]     mounter_test.go:77: 
[2022-06-14T01:22:07.009Z]                 Error Trace:        mounter_test.go:77
[2022-06-14T01:22:07.009Z]                 Error:              Should be true
[2022-06-14T01:22:07.009Z]                 Test:               TestResolvedTs
[2022-06-14T01:22:07.009Z] [2022/06/14 09:16:14.901 +08:00] [INFO] [changefeed.go:354] ["handle incompatible protocol from sink URI"] [oldUriQuery="protocol=default&max-message-size=15"] [fixedUriQuery="max-message-size=15&protocol=open-protocol"]
[2022-06-14T01:22:07.009Z] 
[2022-06-14T01:22:07.009Z] DONE 2494 tests, 1 failure in 534.720s
[2022-06-14T01:22:08.392Z] make: *** [unit_test_in_verify_ci] Error 1
```

### Anything else we need to know

- Does this test exist for other branches as well?

- Has there been a high frequency of failure lately?

--- a/cdc/model/mounter_test.go
+++ b/cdc/model/mounter_test.go
@@ -52,8 +52,8 @@ func TestResolvedTs(t *testing.T) {
 	invalidResolvedTs := ResolvedTs{Mode: -1, Ts: 1}
 	require.Equal(t, uint64(0), invalidResolvedTs.ResolvedMark())
 
-	ts := rand.Uint64()
-	batchID := rand.Uint64()
+	ts := rand.Uint64()%10 + 1
+	batchID := rand.Uint64()%10 + 1
 	normalResolvedTs := NewResolvedTs(ts)
 	batchResolvedTs1 := ResolvedTs{Mode: BatchResolvedMode, Ts: ts, BatchID: batchID}
 	require.True(t, normalResolvedTs.EqualOrGreater(batchResolvedTs1))
@@ -67,14 +67,16 @@ func TestResolvedTs(t *testing.T) {
 	require.True(t, batchResolvedTs2.Less(normalResolvedTs))
 	require.True(t, batchResolvedTs1.Less(batchResolvedTs2))
 
-	largerResolvedTs := NewResolvedTs(ts + rand.Uint64()%10)
+	largerTs := ts + rand.Uint64()%10 + 1
+	largerResolvedTs := NewResolvedTs(largerTs)
 	require.True(t, largerResolvedTs.EqualOrGreater(normalResolvedTs))
 	largerBatchResolvedTs := ResolvedTs{
 		Mode:    BatchResolvedMode,
-		Ts:      ts + rand.Uint64()%10,
+		Ts:      largerTs,
 		BatchID: batchID,
 	}
-	require.True(t, largerBatchResolvedTs.EqualOrGreater(normalResolvedTs))
+	require.True(t, largerBatchResolvedTs.EqualOrGreater(normalResolvedTs),
+		"largerBatchResolvedTs:%+v\nnormalResolvedTs:%+v", largerBatchResolvedTs, normalResolvedTs)
 
 	smallerResolvedTs := NewResolvedTs(0)
 	require.True(t, normalResolvedTs.EqualOrGreater(smallerResolvedTs))

